optaplanner-docs/src/main/asciidoc/Chapter-Quick_start/Chapter-Quick_start.adoc

```diff
@@ -445,14 +467,18 @@ public class CloudBalancingEasyScoreCalculator implements EasyScoreCalculator<Cl
 ----
 ====
 
-Even if we optimize the code above to use ``Map``s to iterate through the `processList` only once, *it is still slow* because it does not do <<incrementalScoreCalculation,incremental score calculation>>.
+Even if we optimize the code above to use ``Map``s to iterate through the `processList` only once, it is still slow because it does not do <<incrementalScoreCalculation,incremental score calculation>>.
```

Comment:
Please keep the boldness of "it is still slow". I've seen users claim optaplanner doesn't scale while using an easy score calculator because they don't realize that that is the problem. It needs to stand out.